app/views/tag/contributors.html.erb

```diff
@@ -57,8 +57,9 @@
   <% end %>
   <hr />
 
-  <% if @notes.nil? || @notes.length == 0 %>
```

Comment:
Thanks! I think perhaps we still need this line in case there is no wildcard but also no contributors. But it could be moved under the `if @wildcard` condition into an `else if @notes.nil? || @notes.length == 0` conditional -- how does that sound?